src/Datadog.Trace.OpenTracing/HttpHeadersCodec.cs

```diff
@@ -42,7 +42,7 @@ public void Inject(global::OpenTracing.ISpanContext context, object carrier)
             else
             {
                 // any other OpenTracing.ISpanContext
-                headers.Set(HttpHeaderNames.TraceId, context.TraceId.ToString(InvariantCulture));
+                headers.Set(HttpHeaderNames.TraceId, context.TraceId);
```

Comment:
Can we emit the string as decimal representation here to maintain the same behavior?

Reply:
Following the https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md this should be hex formatted, I can change it so that for 64bit version the decimal is returned to preserve previous datadog behaviour. Would that be ok for you?